Doctrine/Phpcr/SitemapDocumentProvider.php

```diff
@@ -42,7 +42,7 @@ public function load($sitemap)
     {
         // todo rewrite query when https://github.com/symfony-cmf/CoreBundle/issues/126 is ready
         $documentsCollection = $this->manager->createQuery(
-            'SELECT * FROM [nt:unstructured] WHERE (visible_for_sitemap = true)',
+            'SELECT * FROM [nt:unstructured] WHERE (visible_for_sitemap = true) ORDER By id',
```

Comment:
it does make sense to order the pages by path, i am just not sure if
PHPCR can do it, or if it can if this is the way to achieve it.